contentcuration/contentcuration/views/admin.py

```diff
@@ -141,6 +142,149 @@ def get_all_users(request):
     return Response(user_serializer.data)
 
 
+class ChannelUserListPagination(PageNumberPagination):
+    page_size = 500
+    page_size_query_param = 'page_size'
+    max_page_size = 500
+
+AND_ = lambda x: 'and_'+x
```

Comment:
Here is where the logical operator syntax could be changed.  It could make sense to have `&` be the default, since it will probably be used the most.  So the following example would find channels that are both published AND created by ricecooker:
```
GET /api/get_channels/?is_published=True&is_ricecooker=True
```
To make `&` the default, it'd just be a matter of changing this line to:
```
AND_ = lambda x: x
```